### ISSUE TITLE
Intro to CSS: <CSS Exercises: Foundations, 02-class-id-selectors Issue>

### ISSUE THREAD

**Issue by codingcrumpet** [TheOdinProject/css-exercises#457]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

I just completed this exercise but noticed an issue with the provided .png screenshot. In it, you can see that all classes/divs are using the fonts requested in the "README", but when you open "solution.html" you'll see that the 2 ids (#two and #four) are still defaulting to browser's default font. I think the solution.html file should either include the addition of "font-family" to the 2 ids, or change the .png file so that it reflects the solution.html.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-intro-to-css

### (Optional) Discord Name

sassycrumpet

**Comment by wise-king-sullyman** [comment]
Thanks for putting this issue together!

I don't have a great eye for fonts and might be missing something here, but it looks like in the spec it says only the odd numbers should have a custom font?

2 and 4 don't look like any of the fonts listed to me, what they will look like locally will I think depend on the default font of each users browser.

If I seem to be missing something here though let me know.

**Comment by codingcrumpet** [comment]
My sincerest apologies - I completely forgot that I had registered to GitHub with a new email address, so I didn't see any notifications until today. You are also entirely correct regarding the font differences coming from using different browsers, and the note in the README also explains that.  I think that maybe in the note it could be slightly altered to "Your browsers font's, especially for the second and fourth element, might be different than the one displayed in the desired outcome image." because that would prevent any confusion regarding those two lines. It seems obvious (especially now after being on the final section of CSS), but I saw a number of mentions on this issue popping up in discord as well. I think the beginner's logic is that "Oh, the fonts aren't matching, so I'm doing something wrong."

Thank you again for addressing my comment/issue and have a wonderful day.
